<ui version="4.0" >
 <class>ProjectManager</class>
 <widget class="QWidget" name="ProjectManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>598</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Ignored" hsizetype="Ignored" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Project Manager</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>Project Manager</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newProjectButton" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>90</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeProjectButton" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>90</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addFileButton" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>90</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeFileButton" >
   <property name="geometry" >
    <rect>
     <x>410</x>
     <y>90</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openFileButton" >
   <property name="geometry" >
    <rect>
     <x>520</x>
     <y>90</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget" >
   <property name="windowModality" >
    <enum>Qt::WindowModal</enum>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>130</y>
     <width>237</width>
     <height>457</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editTriggers" >
    <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="dragDropMode" >
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="viewMode" >
    <enum>QListView::ListMode</enum>
   </property>
   <property name="currentRow" >
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Projects</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="fileListWidget" >
   <property name="windowModality" >
    <enum>Qt::WindowModal</enum>
   </property>
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>130</y>
     <width>365</width>
     <height>455</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="dragDropMode" >
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>420</x>
     <y>60</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Files</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>80</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2" >
   <property name="geometry" >
    <rect>
     <x>390</x>
     <y>80</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
